### ISSUE TITLE
Create LocalContext Examples

### ISSUE THREAD

**Issue by pbuttigieg** [iodepo/odis-in#11]
Building on https://github.com/iodepo/odis-arch/issues/398

Including
* Projects
* Datasets
* Places

@arojas1

**Comment by jmckenna** [comment]
@pbuttigieg I don't see the pull request here yet

**Comment by pbuttigieg** [comment]
No, it's just a branch for now. Don't think we're ready to initiate a PR. we can when we have the other types drafted .

**Comment by jmckenna** [comment]
Commit was https://github.com/iodepo/odis-in/commit/62185a191d2117b63c2f7d514872b7bb683a7b2f (in the commit message if you enter `issue #11` it would have tagged it here magically).

**Comment by pbuttigieg** [comment]
> Commit was https://github.com/iodepo/odis-in/commit/62185a191d2117b63c2f7d514872b7bb683a7b2f (in the commit message if you enter `"... (issue #11)"` it would have tagged it here magically).

this issue is linked to the branch where all commits will occur. We don't need to, and shouldn't, cross-reference issues / PRs in each commit message to avoid clutter. exceptions only when there's a need to point to a specific comment in the issue thread .

**Comment by jmckenna** [comment]
@pbuttigieg tagging commits to issues (or through a pull request) help discover those commits easily later on (I can understand for yourself, as you made the commit, but months from now that commit would be impossible to find, as no one else would know what branch it lives in).

**Comment by jmckenna** [comment]
(explained by PL offline, I apologize for my questions here).  I understand now.

**Comment by pbuttigieg** [comment]
never apologise for questions :) 

For others - the way that GitHub now allows one to auto-link an issue to a branch (named after the issue) negates the need to tag commits, as the whole branch is linked. 

We assume this was implemented to cut down clutter

**Comment by jmckenna** [comment]
Here is the latest changes (minor formatting changes): [local-contexts-project-example.json](https://github.com/iodepo/odis-in/blob/11-create-localcontext-examples/dataGraphs/thematics/projects/graphs/local-contexts-project-example.json)

**Comment by pbuttigieg** [comment]
Thanks @jmckenna - getting closer. We have to figure out what properties map to _each_ label and notice, as they are often different kinds of things
